infrastructure/scripts/release.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Usage:
+## Just pass the name of the env we want to tag and deploy.
+## This will create a tag locally with a format of $$ENV-$BRANCH-$CURRENT_USER-$TIMESTAMP 
+## Then push it to the remote git.
+ENV=$1
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+CURRENT_USER=$(whoami)
+TIMESTAMP=$(date +%d-%m-%y--%H%M%S)
+TAG_NAME="release-$ENV-$BRANCH-$CURRENT_USER-$TIMESTAMP"
+
+echo "Current branch name is" "$BRANCH"
+echo "Current environment name is" "$ENV"
+echo "Timestamp assigned will be $TIMESTAMP"
+echo "New tag name will be " "$TAG_NAME"
+
+if [ $ENV == 'prod' ]; then
+    echo ''
+    if [ $BRANCH != 'main' ]; then
+        echo -e "\033[0;31mYou can only deploy to prod from main branch\033[0m"
+        exit 0
+    fi
+    read -p "DANGER: Are you sure you want to push to prod without a PR? Enter y to continue: " answer
```

Comment:
Think we want to not have this. In general, we want the runners to only deploy to prod from main.